### ISSUE TITLE
Normalization of each output before activation

### ISSUE THREAD

**Issue by daisyleedq** [WojciechMormul/yolo2#10]
From the paper and also other keras implementation of YOLO2, I saw normalization of each layer output. Have you implement this layer or just I missed it somewhere, could you point it out. Thank you a lot

**Comment by WeiiJK** [comment]
Batch normalization is in the function of **conv_layer**.
